frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/user-templates.tsx

```diff
@@ -28,6 +35,40 @@ export const UserTemplates: React.FC<UserTemplatesProps> = React.memo(
     const hasUserTemplates = sortedNames.length > 0;
     const hasFieldValue = typeof iGetFieldValue(userTemplateField) === 'undefined';
 
+    const optionUserTemplate = (
+      <>
+        <FormSelectOption
+          key={userTemplateName}
+          value={userTemplateName}
+          label={userTemplateName}
+        />
+      </>
+    );
+
+    const optionNoTemplate = (
+      <>
+        <FormSelectOption
+          key={SELECT_TEMPLATE}
+          value=""
+          label={SELECT_TEMPLATE}
+          isDisabled={!hasFieldValue}
+        />
+        <FormSelectOption key={NO_TEMPLATE} value="" label={NO_TEMPLATE} />
+      </>
+    );
+
+    const optionNoTemplatesAvailable = (
+      <FormSelectOption
+        key={NO_TEMPLATE_AVAILABLE}
+        value=""
+        label={openshiftFlag ? NO_TEMPLATE_AVAILABLE : NO_OPENSHIFT_TEMPLATES}
+      />
+    );
+
+    const optionsList = sortedNames.map((name) => (
```

Comment:
no good reason to have this extracted IMO

Reply:
right ! fixed